### PR TITLE
Adds unit tests for deleteTabDialogueController [closes #231]

### LINKED ISSUE #231: Delete Tab Dialogue Controller

### PR DESCRIPTION
**Pull request by marcoviappiani**

### CODE REVIEW

File: modules/core/tests/client/deleteTabDialogueController.test.client.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    });
+
+  });
+
+  describe('cancel', function () {
+
+    it('cancel should be a function', function () {
+      expect(typeof vm.cancel).toEqual('function');
+    });
+  });
```
Comment (codydaig) [inline modules/core/tests/client/deleteTabDialogueController.test.client.js:73]: New Line

File: modules/core/tests/client/deleteTabDialogueController.test.client.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  describe('hide', function () {
+
+    it('hide should be a function', function () {
+      expect(typeof vm.hide).toEqual('function');
+    });
+  });
```
Comment (codydaig) [inline modules/core/tests/client/deleteTabDialogueController.test.client.js:81]: New Line

File: modules/core/tests/client/deleteTabDialogueController.test.client.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+  describe('tab', function () {
+
+    it('tab should be an object', function () {
+      expect(typeof vm.tab).toEqual('object');
+    });
```
Comment (codydaig) [inline modules/core/tests/client/deleteTabDialogueController.test.client.js]: New Line

File: modules/core/tests/client/deleteTabDialogueController.test.client.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+    it('tab should equal the passed in tab', function () {
+      expect(vm.tab).toEqual(tab);
+    });
+  });
```
Comment (codydaig) [inline modules/core/tests/client/deleteTabDialogueController.test.client.js:93]: New Line

File: modules/core/tests/client/deleteTabDialogueController.test.client.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+  describe('saveAction', function () {
+
+    it('saveAction should be a function', function () {
+      expect(typeof vm.saveAction).toEqual('function');
+    });
+  });
```
Comment (codydaig) [inline modules/core/tests/client/deleteTabDialogueController.test.client.js:101]: New Line

File: modules/core/tests/client/deleteTabDialogueController.test.client.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+});
```
Comment (codydaig) [inline modules/core/tests/client/deleteTabDialogueController.test.client.js:103]: New Line

**Comment by codydaig** [comment]
@marcoviappiani Made some small line comments. Your spacing if off slightly with new lines. Once those are fixed it looks good to me.

**Comment by marcoviappiani** [comment]
addressed comments. let me know if there is anything else!

**Pull request merged**